satellite/attribution/db.go

```diff
@@ -23,10 +23,23 @@ type Info struct {
 	CreatedAt  time.Time
 }
 
+// ValueAttributionRow represents data from QueryValueAttribution without exposing dbx
+type ValueAttributionRow struct {
+	PartnerID          []byte
+	ProjectID          []byte
+	BucketName         []byte
+	RemoteBytesPerHour float64
+	InlineBytesPerHour float64
+	EgressData         int64
+	LastUpdated        time.Time
```

Comment:
I think its not used anywhere.

Reply:
These are used in cmd/satellite/reports/attribution.go when generating the report

Reply:
hmm but its not scanned from DB row here https://github.com/storj/storj/pull/2288/files#diff-b6124f33688583621d38fab4987d6cc0R177

Reply:
Ahhh.  You are just referring to the LastUpdated.  Removed.